modules/govuk/manifests/apps/email_alert_api/sidekiq_queue_check.pp

```diff
@@ -0,0 +1,37 @@
+# == Define: govuk::apps::email_alert_api::sidekiq_queue_check
+#
+# Creates an Icinga check which monitors a sidekiq queue.
+#
+# === Parameters
+#
+# [*ensure*]
+#   Whether to enable the check.
+#   Default: present
+#
+# [*title*]
+#   The name of the sidekiq queues to look for.
+#
+# [*latency_warning*]
+#   The warning threshold for a sidekiq queue in minutes.
+#   Default: 5
+#
+# [*latency_critical*]
+#   The critical threshold for a sidekiq queue in minutes.
+#   Default: 10
+#
+define govuk::apps::email_alert_api::sidekiq_queue_check (
+  $ensure = present,
```

Comment:
`$ensure` is unused here.

You could pass it through to the `icinga::check::graphite` resource, but that isn't actually necessary as Puppet will automatically remove old configuration. I'd perhaps just remove it as a parameter.